cluster/v1alpha1/types.go

```diff
@@ -233,6 +233,10 @@ type PlacementSpec struct {
 	// Predicates represent a slice of predicates to select ManagedClusters. The predicates are ORed.
 	// +optional
 	Predicates []ClusterPredicate `json:"predicates,omitempty"`
+
+	// PrioritizerConfigs override the default configuration of the prioritizers
```

Comment:
describe what empty means.  Is there a difference between empty and nil?

If I have no preference, is there some kind of  default that I can get?

Reply:
If I express some preferences for say balance and steady, and a new prioritizer like resourceratioCPU comes along, what happens?

Reply:
1. For the empty and nil question. 
If no PrioritizerConfigs defined(empty and nil), the placement will use default Prioritizer setting. That is "Steady" with weight 1 and "Balance" with weight 1. Will update the doc.
2. Is the second question an upgrade case? 
I suppose for any new prioritizer added in the future, if they are not in the default setting, user needs to update the placement to with new prioritizer explicitly. 
/cc @elgnay @qiujian16